src/components/Buyer/CarbonCredit.js

```diff
@@ -1,7 +1,20 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import "./CarbonStyle.css"
 
-function CarbonCredit() {
-  return <div>CarbonCredit</div>;
+function CarbonCredit(props) {
+  return (
+  <>
+  {props.content.map((carbons)=>(
+      <div className="Card">
+      <img src={carbons.image} alt="image"/>
```

Comment:
-Download this image https://unsplash.com/photos/nTRDnDdDYk8 and set is as the default instead of carbons.image